### PR TITLE
Add user-agent header with version

### PR DESCRIPTION
**Pull request by timoinstana**
# Why

In order to provide better support for Instana release integrations we would like to know about the user agent which sends API requests (including the version). This also helps to get a better understanding of adaption.

# What

Add a simple version file based on the latest git tag which is used for the `curl` requests. The latest version is also fetched by a simple `curl`.
In addition there is a test with https://github.com/nektos/act to test execution locally.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,6 +3,8 @@ FROM alpine:3.10
 
 RUN apk add --no-cache curl jq
 
+RUN curl --silent https://api.github.com/repos/taimos/github-action-instana-release/releases/latest | jq -r '.tag_name' > /.version
```
Comment (mmanciop) [inline Dockerfile]: IMO this should come as a build arg

**Review by mmanciop — CHANGES_REQUESTED** [review]
IMO the version should come in as a build arg, the curl inside the Dockerfile leads to wrong versions when building the image locally

File: Dockerfile (hunk 1 above)
Comment (timoinstana) [reply Dockerfile]: I agree but couldn't find a way to get that information into the GH action.

**Comment by hoegertn** [comment]
This looks good to me. Is this final and can be merged and released?

**Comment by mmanciop** [comment]
Wait I have a small improvement :-)

File: Dockerfile (hunk 1 above)
Comment (mmanciop) [reply Dockerfile]: Solved in https://github.com/mmanciop/github-action-instana-release/commit/e0dd716df01f5665e46821fd6273a1e465517aaa

**Comment by mmanciop** [comment]
@hoegertn it should be good now.

**Review by mmanciop — APPROVED** [review]

**Pull request merged**